modules/ROOT/pages/billing/usage-report.adoc

```diff
@@ -1,72 +1,88 @@
-= Usage report
+= Billing and usage report
 :description: usage reporting allows customers to monitor their billing and credit consumption.
 :page-role: aura-db-professional aura-db-business-critical aura-db-dedicated
 
-The usage report gives real-time insights into resource usage for the current project, including both running and paused states.
+== Required roles
 
-The default time frame is month-to-date.
-The usage report is updated hourly.
+* The usage report is available to `Organization Owner`, `Organization Admin`, and `Project Admin`. The usage report can be viewed at either the organization-level (all projects) or the project-level (a single project). 
+* Billing is available to `Organization Owner`. 
+Billing information is only available at the organization-level.
```

Comment:
```suggestion
The usage report is available to `Organization Owner`, `Organization Admin`, and `Project Admin`. The usage report can be viewed at either the organization-level (all projects) or the project-level (a single project). 
Billing is available to `Organization Owner`. 
Billing information is only available at the organization-level.
```
I would suggest to rewrite this to running text as an introduction instead of tarting the page with a bullet list. Briefly explain the hierarchy, ie Billing is available at org level but you can see usage either for the whole org or by project.